tests/test_edgeql_policies.py

```diff
@@ -1296,3 +1296,47 @@ async def test_edgeql_policies_empty_cast_01(self):
             __typenames__=True,
         )
         self.assertEqual(obj, [])
+
+
+class TestEdgeQLPoliciesStandalone(tb.QueryTestCase):
```

Comment:
I'd make this a ddl or data_migration test instead of creating a new test suite for it. New test suites have some setup costs.